app/web/modules/discrepancies/resolution.njk

```diff
@@ -0,0 +1,24 @@
+{% extends "layout/layout.njk" %}
+{% from "components/button/macro.njk" import govukButton %}
+
+{% block main %}
+  <h1 class="govuk-heading-m">Discrepancy Resolution</h1>
```

Comment:
To save @jonheslop from having to make a PR every time we do I think the GOV.UK design system favours sentence case for headings https://design-system.service.gov.uk/styles/typography/#headings

as per https://github.com/alphagov/pay-toolbox/pull/11